### PR TITLE
docs: Add missing line in *zone-flags* file. Original referenced from [now achieve](angular.io/guide/user-input#source-code)

### PR DESCRIPTION
**Pull request by aakash14goplani**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

- Current document misses the statement that is supposed  to be added in custom zone file for supporting passive event listeners. 

Issue Number: N/A

## What is the new behavior?
- Added that line by referencing [archives](angular.io/guide/user-input#source-code)

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/event-binding.md [hunk 1]
```diff
@@ -50,7 +50,10 @@ Angular also supports passive event listeners. For example, use the following st
 
 1. Create a file `zone-flags.ts` under `src` directory.
 2. Add the following line into this file.
-3. In the `src/polyfills.ts` file, before importing zone.js, import the newly created `zone-flags`.
+   ```
+   (window as any)['__zone_symbol__PASSIVE_EVENTS'] = ['scroll'];
+   ```
+4. In the `src/polyfills.ts` file, before importing zone.js, import the newly created `zone-flags`.
```
Comment (AndrewKushnir) [inline aio/content/guide/event-binding.md:56]:
```suggestion
3. In the `src/polyfills.ts` file, before importing zone.js, import the newly created `zone-flags`.
```

File: aio/content/guide/event-binding.md [hunk 2]
```diff
@@ -50,15 +50,14 @@ Angular also supports passive event listeners. For example, use the following st
 
 1. Create a file `zone-flags.ts` under `src` directory.
 2. Add the following line into this file.
-   ```
+   ```js
```
Comment (AndrewKushnir) [inline aio/content/guide/event-binding.md:53]:
```suggestion
   ```typescript
```

**Review by AndrewKushnir — COMMENTED** [review]
@aakash14goplani thanks for changing the item number. I noticed that you've also added `js` to the code snippet wrappers, could you please use `typescript` instead?

One additional comment: once the changes are made, please merge all commits into one and let us know (as a comment in this PR), so we can do a final review.

Thank you.

File: aio/content/guide/event-binding.md (hunk 1 above)
Comment (aakash14goplani) [reply aio/content/guide/event-binding.md]: Done. Thanks for pointing that out :)

File: aio/content/guide/event-binding.md (hunk 2 above)
Comment (aakash14goplani) [reply aio/content/guide/event-binding.md]: Thanks for suggestion, working on squashing the comments into one. Thanks

File: aio/content/guide/event-binding.md (hunk 1 above)
Comment (aakash14goplani) [reply aio/content/guide/event-binding.md]: Squashed multiple comments into one!

File: aio/content/guide/event-binding.md (hunk 2 above)
Comment (aakash14goplani) [reply aio/content/guide/event-binding.md]: Squashed multiple comments into one!

**Review by AndrewKushnir — APPROVED** [review]
@aakash14goplani thanks for addressing the feedback and improving Angular docs 👍

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 2509d7681bc0a4fad8377164ecca3f263a6b44bd.

**Pull request closed without merging**